openprescribing/frontend/tests/test_bookmark_views.py

```diff
@@ -1,15 +1,8 @@
-import base64
-from mock import patch
```

Comment:
Almost nothing makes me happier than seeing

```
- from mock import patch
```

in red in a diff.

Reply:
:) It wasn't even used